Hymns/SceneDelegate.swift

```diff
@@ -19,7 +19,13 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
 
         // Create the SwiftUI view and set the context as the value for the managedObjectContext environment keyPath.
         // Add `@Environment(\.managedObjectContext)` in the views that will need the context.
+        #if !DEBUG
+        let rootView = LaunchRouterView().environment(\.managedObjectContext, context)
+        #endif
+
+        #if DEBUG
         let rootView = HomeContainerView().environment(\.managedObjectContext, context)
+        #endif
```

Comment:
What's this for?

Reply:
This keeps the lottie animation from ever running in debug mode at all. That is what kept breaking all of our tests like anything that was time based. This keeps it from breaking things like ui tests, probably snapshot tests too. So we only run the Lottie animation when not in debug mode

Reply:
Ahh, I didn't see the "!" in fromt of the DEBUG. That's cool!